app/ui-flextab/client/tabs/userActions.js

```diff
@@ -99,6 +99,19 @@ export const getActions = ({ user, directActions, hideAdminControls }) => {
 		}
 	};
 
+
+	const hasAlreadyFollowed = (username) => {
```

Comment:
How about we name it / make a generic "canFollow" method? Which can be later extended to support other use cases (if any)?